common/views/components/IIIFSearchWithin/IIIFSearchWithin.tsx

```diff
@@ -126,16 +124,16 @@ const IIIFSearchWithin: FunctionComponent<Props> = ({
         <SearchInputWrapper className="relative">
           <TextInput
             id={'searchWithin'}
-            label={'Search within this item'}
-            placeholder={'Search within this item'}
+            label={'Enter keyword'}
+            placeholder={'Enter keyword'}
```

Comment:
Is this text going to make sense in the accessibility tree? Keyword for what? Does the header in the sidebar make this make sense? Maybe aria-describedby could be useful.

Reply:
This is a good point, since we style our `label`s as if they were `placeholder`s.